test/Response/JsonResponseTest.php

```diff
@@ -139,7 +140,7 @@ public function testConstructorRewindsBodyStream(): void
         $json     = ['test' => 'data'];
         $response = new JsonResponse($json);
 
-        $actual = json_decode($response->getBody()->getContents(), true);
+        $actual = json_decode($response->getBody()->getContents(), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
This is the first change I'm not entirely clear on. I cannot remember what that third argument is supposed to be so seeing the 512 value makes me scratch my head. That said having an exception here makes a lot of sense since it will ensure that the assertion never gets run. I wonder if it can use named arguments here instead? Perhaps @samsonasik can answer that.

Reply:
512 is the default value. Named parameters were introduced in PHP 8.0, so curious why it didn't use them for this. I guess this demonstrates that rector may be a start but it will nearly always required manual intervention?

Reply:
Yes, it certainly always needs human review

Reply:
I think `JSON_THROW_ON_ERROR` is fine here as this is test class

Reply:
named arg currently used on rector for attribute creation